src/main/scala/com/scalapenos/riak/internal/RiakHttpClientHelper.scala

```diff
@@ -271,6 +283,29 @@ private[riak] class RiakHttpClientHelper(system: ActorSystem) extends RiakUriSup
       }
     }
   }
+
+  private def toRiakSiblingValues(response: HttpResponse): Set[RiakValue] = {
+    import spray.http._
+    import spray.httpx.unmarshalling._
+    import FixedMultipartContentUnmarshalling._
+
+    implicit val FixedMultipartContentUnmarshaller = multipartContentUnmarshaller(HttpCharsets.`UTF-8`)
+
+    val vclockHeader = response.headers.find(_.is(`X-Riak-Vclock`.toLowerCase)).toList
+
+    response.entity.as[MultipartContent] match {
+      case Left(error) ⇒ throw new ConflictResolutionFailed(error.toString)
```

Comment:
Is this the correct exception to throw here? No conflict resolution was attempted so it can't have failed.